core/execution/market.go

```diff
@@ -1964,19 +1947,14 @@ func (m *Market) resolveClosedOutParties(ctx context.Context, distressedMarginEv
 		size = uint64(-networkPos)
 	}
 
-	ref := "LS"
-	if orderID != nil {
-		ref = fmt.Sprintf("LS-%s", *orderID)
```

Comment:
This was just the `m.idgen.NextID()` pulled for a dummy order object passed into the old confirmMTM signature, so hopefully not too controversial...